src/components/SudoApplicationModal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Button, Modal } from "../components";
+
+type Props = {
+  isModalOpen: boolean;
+  setIsModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  address: string;
+};
+
+const SudoApplicationModal: React.FC<Props> = (props) => {
+  const { isModalOpen, setIsModalOpen, address } = props;
+
+  const sendApplication = async () => {
+    window.location.replace(
```

Comment:
Might make more sense for this link to be in an `<a>` tag

Reply:
Wanted to keep the button styling without rebuilding it for an <a> tag - makes no end user difference